.eslintrc.json

```diff
@@ -76,7 +76,7 @@
     "require-yield": "off",
     "prefer-template": "warn",
     "no-underscore-dangle": "off",
-    "comma-dangle": ["warn", "never"],
+    "comma-dangle": ["warn", "only-multiline"],
```

Comment:
This feels like it should be discussed before making this change without prelude.